concent_api/core/message_handlers.py

```diff
@@ -956,34 +957,33 @@ def store_subtask(
 def handle_messages_from_database(client_public_key: bytes) -> Union[message.Message, None]:
     assert client_public_key not in ['', None]
     encoded_client_public_key = b64encode(client_public_key)
-
     with non_nesting_atomic(using='control'):
-        pending_response = PendingResponse.objects.select_for_update().filter(
+        payments_info_responses = PendingResponse.objects.select_for_update().filter(
```

Comment:
It is actually one `response` because you do `first()` at the end of the query.

Reply:
Done